content/en/entities/Instance.md

```diff
@@ -169,7 +169,7 @@ aliases: [
 
 ### `domain` {#domain}
 
-**Description:** The domain name of the instance.\
+**Description:** The domain name of the instance. Corresponds to the configured `LOCAL_DOMAIN`, which is used for account handles, and is not necessarily the same as `WEB_DOMAIN` where the Mastodon API and web interface are hosted.\
```

Comment:
I was going to suggest adding: 
```
If `LOCAL_DOMAIN` is not set, then this will be the host portion of `WEB_DOMAIN`
```

But I'm not sure that's actually true from what I can see in the code?

Reply:
I don't understand what you mean here, but there is not necessarily any relation between `LOCAL_DOMAIN` and `WEB_DOMAIN`